### ISSUE TITLE
an option to disable case-sensitivity in element hiding?

### ISSUE THREAD

**Issue by nimasaj** [uBlockOrigin/uAssets#834]
#### URL(s) where the issue occurs:
`http://trainbit.com/files/200696239884/`

#### Describe the issue:
Is it possible to set an option to disable case-sensitivity in element hiding section (CSS selectors)?
Currently if you define a rule as ####adv5, it won't block Adv5 IDs. And by defining multiple similar rules (playing with case sensitive letters, ADV5,Adv5,adv5,aDV5, ....), you'll end up a huge filter list.

#### Screenshot(s):
`https://imgur.com/a/2AoTn`

#### Versions:
- Browser/version: 62.0.3202.89 (Official Build) (64-bit)
- uBlock Origin version: 1.14.18

**Comment by Hrxn** [comment]
I don't think that this is possible.. Selector matching is done by the browser engine itself.
